app/views/news/2018-11-14-announcing-project-proposals.html.md

```diff
@@ -0,0 +1,11 @@
+---
+summary: Ruby Together is now accepting proposals for quarterly project funding from any Ruby OSS project.
+---
+
+This update doesn't fit exactly into our regular monthly news cycle, but we are excited to announce [project proposals](/projects).
+
+Ruby Together is now accepting proposals from Ruby OSS projects for three months of funded work at the rate of $10,000 USD per month.
```

Comment:
The wording here is a little weird. 

Ruby Together is now accepting proposals from Ruby OSS projects for three months of funded work. The total amount of funding that can be requested is $10,000.

Reply:
It would be great to add a little reasoning here as well. Like in order to make it easier for anyone to be a ruby together project... or something like that.